src/Functions/return123.cpp

```diff
@@ -0,0 +1,46 @@
+#include <Functions/FunctionFactory.h>
```

Comment:
The code here is just a dummy, right?

Reply:
Yes, I tested how to add a function using this code, this is useless, I will delete it